src/middlewared/middlewared/plugins/catalogs_linux/sync_catalogs.py

```diff
@@ -70,3 +74,12 @@ def update_git_repository(self, catalog):
         return pull_clone_repository(
             catalog['repository'], os.path.dirname(catalog['location']), catalog['branch'],
         )
+
+    @private
```

Comment:
We have a @cache decorator that can be used so we don't have to create a class attribute